viewing_party/party.py

```diff
@@ -0,0 +1,186 @@
+# Wave 1
+def create_movie(title, genre, rating):
+    # title = str, genre = str, rating = float
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict['title'] = title
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+    else:
+        return None
+    return movie_dict
```

Comment:
This works great but you could just return the title. genre, and rating without doing movie_dict = {}
```suggestion
def create_movie(title, genre, rating):
    if title == None or genre == None or rating == None:
        return None
    else:
        return {
        "title":title, 
        "genre":genre, 
        "rating":rating
        }
```